src/server/query.c

```diff
@@ -65,6 +66,58 @@ tds_lastpacket(TDSSOCKET * tds)
 	return tds->in_buf[1] != 0;
 }
 
+/**
+ * get query packet of a given type
+ * \tds
+ * \param head         extra information to put in a TDS7 header
+ */
+static TDSRET
+tds_get_query_head(TDSSOCKET * tds, TDSHEADERS * head)
+{
+	if (!IS_TDS72_PLUS(tds->conn)) {
+		return TDS_SUCCESS;
+	}
+
+	int qn_len = 0, more;
```

Comment:
we support very old systems that does not like to mix declaration and statements, declarations should be at the beginning of blocks.

Reply:
moved to after declarations